pgpool2-4.5.yaml

```
@@ -0,0 +1,109 @@
+package:
+  name: pgpool2-4.5
+  version: 4.5.4
+  epoch: 0
+  description: Middleware that works between PostgreSQL servers and a PostgreSQL database client.
+  copyright:
+    - license: BSD-3-Clause AND MIT
+  dependencies:
+    provides:
+      - pgpool2=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - flex
+      - libtool
+      - openssl-dev
+      - postgresql-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: mangled-package-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/pgpool/pgpool2
+      tag: V${{vars.mangled-package-version}}
+      expected-commit: 54ccb32a3a09fb9984329562db430a96e7b8478c
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --sysconfdir=/etc/pgpool2 --with-openssl
+
+  # build src/parser explicitly to avoid build failures due to missing parser files with -j flag
+  - runs: |
+      make -C src/parser V=1
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: pgpool2-4.5-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      provides:
+        - pgpool2-dev=${{package.full-version}}
+      runtime:
+        - pgpool2-4.5
+
+  - name: pgpool2-4.5-bitnami-compat
```

```suggestion
  - name: ${{package.name}}-bitnami-compat
```

We seems to have settled on this convention for compat package names. So it would be a similar change for the dev package too

---

Thanks. Updated.